trade_remedies_public/templates/registration/accessibility_statement.html

```diff
@@ -4,72 +4,129 @@
-                <p>
-                    <p>This statement was prepared on 03 July 2019 It was last updated on 03 July 2019.</p>
-                    <h2 class="heading-medium">How we tested this website</h2>
-                    <p>This website was last tested on 16 April 2019. The test was carried out by a DIT appointed independent
-                        specialist, <a href="http://www.digitalaccessibilitycentre.org">Digital Accessibility Centre</a>.</p>
-
-                    <p>Evidence for this accessibility assessment:</p>
-                    <ul class="list list-bullet">
-                        <li>we have supplied alt text for images</li>
-                        <li>the tables follow the <a href="https://design-system.service.gov.uk/">GDS design system</a> so have
-                            already been tested for accessibility compliance
-                        </li>
-                        <li>the content structure is in a logical order and broken up with clear headers and sub headers and
-                            form fields are labelled.
-                        </li>
-                        <li>the pages can be navigated using a keyboard with focus on tab</li>
-                        <li>text and graphic elements have sufficient contrast against the background</li>
-                        <li>the page title indicates the website’s purpose and content</li>
-                        <li>the purpose of every link is clear from the link text</li>
-                        <li>the website uses valid HTML</li>
-                        <li>the website is content so interactive elements are limited to links, with graphics limited to
-                            graphs
-                        </li>
-                        <li>users can change the text spacing or line height in their browser</li>
-                        <li>the service was tested using (and works with) the following assistive tools: ZoomText, Jaws, NVDA,
-                            VoiceOver, DragonVoice, Talkback.
-                        </li>
-                    </ul>
-
-                </div>
-
-                <div class="column-one-third right-column">
-                    {% block right_column %}
-                    {% endblock %}
-                </div>
-            </div>
+            <p>We’ll consider your request and get back to you in 5 days.</p>
+
+            <h2 class="heading-medium">Reporting accessibility problems with this service</h2>
+            <p>We’re always looking to improve the accessibility of this service. If you find any problems 
+                that are not listed on this page or think we're not meeting accessibility requirements:</p>
+            <ul class="list list-bullet">
+                <li>email <a href="mailto: contact@traderemedies.gov.uk">contact@traderemedies.gov.uk</a></li> {# /PS-IGNORE #}
+                <li>call +44 (0) 20 7215 5000</li>
```

Comment:
Shall we make this a clickable link?

```suggestion
               <li><a href="tel:+442072155000">call +44 (0) 20 7215 5000</a></li>
```

Reply:
Resolved in https://github.com/uktrade/trade-remedies-public/pull/244/commits/b9acf08f1b828334803a23ceb7d8ce1a4dd8380a.